scripts/apps/archive/index.js

```diff
@@ -198,6 +199,26 @@ angular.module('superdesk.apps.archive', [
                 groupLabel: gettext('Duplicate'),
                 groupIcon: 'copy'
             })
+            .activity('label', {
+                label: gettext('Set label in current package'),
+                priority: 30,
+                icon: 'label',
+                monitor: true,
+                disableInList: true,
```

Comment:
this should be probabaly `list: false` to be consistent with other options like monitor and dropdown